arm-datalake-store/filesystem/2016-11-01/swagger/filesystem.json

```diff
@@ -868,14 +969,61 @@
           },
           {
             "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
           },
+          "default": {
+            "description": "An unexpected error from the server.",
+            "schema":
+            {
+              "$ref": "#/definitions/AdlsError"
+            }
+          }
+        }
+      }
+    },
+    "/webhdfs/v1/{defaultAclFilePath}": {
+      "put": {
+        "tags": [
+          "FileSystem"
+        ],
+        "operationId": "FileSystem_RemoveDefaultAcl",
+        "description": "Removes the existing Default Access Control List (ACL) of the specified directory.",
+        "parameters": [
           {
-            "$ref": "#/parameters/SubscriptionIdParameter"
+            "name": "defaultAclFilePath",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Data Lake Store path (starting with '/') of the directory with the default ACL being removed."
+          },
+          {
+            "name": "op",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The constant value for the operation.",
+            "enum": [
+              "REMOVEDEFAULTACL"
+            ]
```

Comment:
same comment about use of single enum == constant in AutoRest. If this needs to change for some reason, please let me know and we can change them.